docs/application/web/guides/w3c/ui/ui-layout.md

```diff
@@ -335,9 +335,9 @@ Tizen supports a special media query feature for a circular device shape. The fo
 
 **Table: Tizen specialized media feature**
 
-| Feature                  | Value         | Min/Max    | Description |
-| ------------------------ | ------------- | ---------- | ----------- |
-| `-tizen-geometric-shape` | `"rectangle"` | `"circle"` | No          |
+| Feature                  | Value                    | Min/Max    | Description  |
+| ------------------------ | ------------------------ | ---------- | ------------ |
+| `-tizen-geometric-shape` | `"rectangle" | "circle"` | No         | Device shape |
```

Comment:
Fill in the table with right value.

Reply:
Because of markdown coding style, "|" inside a table causes an error. So I used "/" instead of "|" in this line.

Reply:
Reverted "/" to "|" as future system can handle this.